packages/lesswrong/components/posts/PostsCommentsThread.tsx

```diff
@@ -21,35 +20,20 @@ const PostsCommentsThread = ({ post, terms, newForm=true, timelineView=false }:
     return <Components.Loading />;
   } else if (!results) {
     return null;
-  } else {
-    const nestedComments = unflattenComments(results);
-    if (timelineView) {
-      return (
-        <Components.CommentsTimelineSection
-          comments={nestedComments}
-          loadMoreComments={loadMore}
-          totalComments={totalCount as number}
-          commentCount={(results && results.length) || 0}
-          loadingMoreComments={loadingMore}
-          loadMoreCount={10}
-          post={post}
-          newForm={newForm}
-        />
-      );
```

Comment:
The two cases have diverged sufficiently that I decided to split this into two files (this and `SubforumCommentsThread`)